bindings/python/pymongoarrow/types.py

```diff
@@ -47,7 +49,8 @@ def _is_objectid(obj):
     _atypes.is_int64: _BsonArrowTypes.int64,
     _atypes.is_float64: _BsonArrowTypes.double,
     _atypes.is_timestamp: _BsonArrowTypes.datetime,
-    _is_objectid: _BsonArrowTypes.objectid
+    _is_objectid: _BsonArrowTypes.objectid,
+    _atypes.is_string: _BsonArrowTypes.string
```

Comment:
Suggest adding a trailing comma to make future diffs nicer :)

Reply:
Done